.github/scripts/install-drgn.sh

```diff
@@ -8,7 +8,7 @@ sudo apt update
 sudo apt install bison flex libelf-dev libdw-dev libomp5 libomp-dev
 
 git clone https://github.com/osandov/drgn.git
-
 cd drgn
+git checkout -b drgn_february 0a6aaaae5d31ed142448f8220e208d65478ec80d
```

Comment:
Yeah as you mentioned this should probably go. I think Prakash got something working that we should be able to integrate soon.